executor.go

```diff
@@ -8,12 +8,8 @@ type Executor interface {
 	Execute(req *Request, re ResponseEmitter, env Environment) error
 }
 
-// Environment is the environment passed to commands. The only required method
-// is Context.
-type Environment interface {
-	// Context returns the environment's context.
-	Context() context.Context
-}
+// Environment is the environment passed to commands.
+type Environment interface{}
```

Comment:
(doesn't break anything in either go-filecoin or go-ipfs)

Reply:
Should `MakeEnvironment` be removed with this?

Reply:
We still need the environment (we type assert it to something more useful).

Really, we should consider stashing everything we'd _usually_ put in the environment in the _context_ (which is, IMO, where it should go) but we can handle that later.